js/sider/LandingssideSide.js

```diff
@@ -16,7 +22,31 @@ import { hentDineSykmeldinger } from '../actions/dineSykmeldinger_actions';
 import { hentLedere } from '../actions/ledere_actions';
 import { hentSykeforloep, hentSykeforloepMetadata } from '../actions/sykeforloep_actions';
 import { skalViseOppfoelgingsdialogLenke } from '../utils/sykmeldingUtils';
+import { skalViseMotebehovMedOppfolgingsforlopListe } from '../utils/motebehovUtils';
 import { hentSoknader } from '../actions/soknader_actions';
+import { hentOppfolgingsforlopsPerioder } from '../actions/oppfolgingsforlopsPerioder_actions';
+import {
+    finnOgHentManglendeOppfolgingsforlopsPerioder,
+    finnOppfolgingsforlopsPerioderForAktiveSykmeldinger,
+    finnVirksomheterMedAktivSykmelding,
+} from '../utils/oppfolgingsforlopsperioderUtils';
+
+export const hentRessurser = (props) => {
+    const {
+        actions,
+        oppfolgingsforlopsPerioderReducerListe,
+        skalHenteMotebehov,
+        skalHenteToggles,
+        virksomhetsnrListe,
+    } = props;
+    if (skalHenteMotebehov) {
```

Comment:
Ville ha flyttet denne sjekken til sagaen. Sagaen har da ansvar for å sjekke om det er nødvendig å gjøre en fetch eller om data allerede er fetchet.